sdk/core/core-rest-pipeline/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## 1.0.4 (Unreleased)
```

Comment:
should this be 1.1 since this is a pretty big new feature? Are we planning to beta it at all?

Reply:
We didn't plan on a beta because the new feature is added in a new policy so the risk of breaking existing libraries is low.  And only ACR is using it (in the form of a private copy) now. KeyVault will move to it later.  Yes we could do a beta.

Reply:
Updated to 1.1.0 beta.1